interface/wx/window.h

```diff
@@ -4099,6 +4099,19 @@ class wxWindow : public wxEvtHandler
     */
     virtual void UpdateWindowUI(long flags = wxUPDATE_UI_NONE);
 
+    /**
+        When UpdateWindowUI() runs, it creates instances of
+        wxUpdateUIEvent.  Those instances should vary depending on the
```

Comment:
I don't think we should be so categoric:
```suggestion
        wxUpdateUIEvent.  Those instances may vary depending on the
```

Reply:
OK